Makefile

```diff
@@ -0,0 +1,11 @@
+#! make
+
+build:
+	docker build -t tsugite .
+
+up:
+	docker-compose up -d
+
+browse:
+	firefox http://localhost:8083 &
```

Comment:
Can we please make this browser-agnostic?

```suggestion
	open http://localhost:8083 &
```

Reply:
xdg-open?

Reply:
Would not work on macOS (Catalina, at least).

Reply:
hmm... `open` on mac, `xdg-open` on linux (without wayland?), `start` on windows... is there a good cross-platform command for launching the default web browser on the system?